client-v2/src/util/configureStore.ts

```diff
@@ -14,17 +14,23 @@ import {
 } from './storeMiddleware';
 import { State } from 'types/State';
 import { ExtraThunkArgs } from 'types/Store';
-import { fetchFrontsConfigStrategy } from 'strategies';
+import { fetchFrontsConfigStrategy } from 'strategies/fetch-fronts-config';
+import { fetchCollectionsStrategy } from 'strategies/fetch-collection';
 
-export default function configureStore(initialState?: State) {
+export default function configureStore(
```

Comment:
Re: conversation, if there's not much in the way of partial application etc. at this level then it might not be necessary to inject with the thunk params 👍

Reply:
💣 they gone